Source/WebKit/_WebKit_SwiftUI/Implementation/CocoaWebViewAdapter.swift

```diff
@@ -32,6 +32,30 @@ typealias CocoaView = NSView
 
 @MainActor
 class CocoaWebViewAdapter: CocoaView, PlatformTextSearching {
+
+#if os(iOS)
+    var geometrySafeAreaInsets: EdgeInsets? = nil  {
```

Comment:
I don't loveee the `geometrySafeAreaInsets` since like safe area insets in general are about the geometry. not sure of a better name though. Maybe like `extrinsicSafeAreaInsets` or something, since they're being propagated from above? @pxlcoder is usually good at naming things like this

Reply:
`extrinsicSafeAreaInsets` sounds good to me!